examples/js/e-commerce-umd/src/widgets/Products.ts

```diff
@@ -51,8 +51,9 @@ export const products = hits<Hit>({
         </article>
       `;
     },
-    empty(results, { html }) {
-      const hasRefinements = results.getRefinements().length > 0;
+    empty(_, { html }) {
+      const hasRefinements =
+        window.search.renderState['instant_search'].clearRefinements?.canRefine;
```

Comment:
normally you can directly read the renderState, but in the layout of this example, the widgets are in standalone files, and we can't import `search` or it would be circular.

Could be interesting in the future if templates got access to InstantSearch instance or parent.